alacritty.yml

```diff
@@ -205,6 +205,9 @@ key_bindings:
 mouse_bindings:
   - { mouse: Middle, action: PasteSelection }
 
+multiclick_threshold_ms: 300
```

Comment:
You've got a point about the mouse config. I would like to keep breaking config changes to a minimum, so it'd be nice to ship something that could potentially be the final arrangement. Maybe something like this:

```yaml
# Configuration for the mouse
# Multi-click thresholds are in units of milliseconds.
mouse:
  double_click: { threshold: 300, action: SelectSemantically }
  triple_click: { threshold: 300, action: SelectLine }

selection:
  semantic_escape_chars: ",│`|:\"' ()[]{}<>"
```

The same config type (`Multiclick`, for example) could be used for both double and triple click sections. I think it would be nice to have a block for `selection` even if there's only one property initially.

We could probably omit the `action` field of the `Multiclick` config for now since making them into an action might be kind of annoying. Adding it later would be backwards compatible with this layout.

I realize this change might be sort of onerous. If you're getting sick of applying these changes, I'd be happy to push commits on top. Just let me know :).